Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: clean
```

Comment:
Let's add this in a different branch/PR. It doesn't really have to do with `list-reports`

Also - can we add it as a git checkout hook?